src/lib/resolvers/message.ts

```diff
@@ -11,9 +11,14 @@ interface MessageResolverOptions {
 	message: Message;
 }
 
+const CHANNEL_MESSAGE_REGEX = /^(?<channelId>\d{17,19})-(?<messageId>\d{17,19})$/;
```

Comment:
This RegExp should be PR'd to `@sapphire/discord.js-utilities`

Reply:
I'll make a PR to `@sapphire/discord-utilities`

Reply:
You can update the yarn lockfile now and import the regex